qa-tests-backend/src/test/groovy/ImageScanningTest.groovy

```diff
@@ -655,8 +655,8 @@ class ImageScanningTest extends BaseSpecification {
         where:
         image                                              | registry
         "registry.k8s.io/ip-masq-agent-amd64:v2.4.1"       | "gcr registry"
-        "docker.io/jenkins/jenkins:lts"                    | "docker registry"
-        "docker.io/jenkins/jenkins:2.220-alpine"           | "docker registry"
+        "quay.io/rhacs-eng/qa:alpine-3.16.0"               | "quay registry"
+        "quay.io/stackrox-io/scanner:2.27.3"               | "quay registry"
```

Comment:
Why do we have two for the same registry (quay)?

Reply:
doesn't really matter much. I just used Quay because we haven't had problems with it before